packages/fx-core/resource/strings.json

```diff
@@ -25,7 +25,7 @@
     "LocalDebugTenantConfirmNotice": "You already run local debug for the Teams app in another tenant '%s'. You can sign out %s and sign in with the matched M365 account to re-use the previous local debug resource. If you really want to switch tenant for local debug, you can delete the localSettings.json file and re-run local debug.",
     "MigrateSuccessNotice": "Your project is now initialized with Teams Toolkit! You can debug your application by pressing F5. If you wish to host your application in Azure, we recommend you to re-create your project directly using the latest Teams Toolkit.",
     "MigrationToArmAndMultiEnvMessage": "Teams Toolkit will migrate your project to support the latest features. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back below Teams Toolkit 2.7.0.",
-    "MigrationToArmAndMultiEnvSuccessMessage": "Migration Success! please reload VSCode and run provision command before executing other commands, otherwise the commands may fail. Provision may create new resource, click the link for more information",
+    "MigrationToArmAndMultiEnvSuccessMessage": "Upgrade Succeeds! We have updated your project with latest Teams Toolkit features and please reload VSCode. Click on Read more to learn why this is needed or how to restore your project",
```

Comment:
I would do it like this:

Upgrade succeeded. Your project is updated with the latest Teams Toolkit features. Restarting Visual Studio Code is required. Click on Read more to learn why this is needed or how to restore your project.

Reply:
Thanks!
Already modified as your provided info.